curriculum/challenges/english/16-the-odin-project/top-basic-function-projects/top-basic-functions-exercise-a.md

```diff
@@ -0,0 +1,61 @@
+---
+id: 6619240f46cec8e04d77e03a
+title: Basic Functions Exercise A 
+challengeType: 1
+dashedName: top-basic-functions-exercise-a
+---
+
+# --description--
+
+Create a function that takes in an integer. This function should return the given interger + 7 if the integer is less than 10, if the integer is greater than or equal to 10, it should return the given integer - 3.
```

Comment:
Small spelling error.
But also, I wonder if it would be better to wrap these expressions in backticks? 🤔 

maybe something like this
```suggestion
Create a function that takes in an integer. This function should return the given `integer + 7` if the integer is less than `10`. If the integer is greater than or equal to `10`, it should return the given `integer - 3`.
```

Reply:
When I initially saw it, I was confused by the +7 not realizing it was supposed to be the expression of integer+7

maybe the addition of backticks will make that more clear

<img width="749" alt="Screenshot 2024-04-16 at 2 30 20 PM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/6286ff5e-c6b1-4409-9ad0-ef46597f2b64">